test-env/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "books",
+  "version": "1.0.0",
+  "main": "server/server.js",
+  "scripts": {
+    "pretest": "jshint ."
+  },
+  "dependencies": {
+    "compression": "^1.0.3",
+    "errorhandler": "^1.1.1",
+    "loopback": "^2.8.0",
+    "loopback-boot": "^2.4.0",
+    "loopback-datasource-juggler": "^2.7.0",
+    "serve-favicon": "^2.0.1"
+  },
+  "optionalDependencies": {
+    "loopback-explorer": "^1.1.0"
```

Comment:
Note: when you upgrade to the current project structure, this will become a regular dependency on `loopback-component-explorer` version `^2.1.1`